[2025-04-01T03:44:25.592+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T03:44:25.608+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.get_age manual__2025-04-01T03:44:24.338278+00:00 [queued]>
[2025-04-01T03:44:25.618+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.get_age manual__2025-04-01T03:44:24.338278+00:00 [queued]>
[2025-04-01T03:44:25.619+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-01T03:44:25.636+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): get_age> on 2025-04-01 03:44:24.338278+00:00
[2025-04-01T03:44:25.643+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=24531) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T03:44:25.645+0000] {standard_task_runner.py:72} INFO - Started process 24534 to run task
[2025-04-01T03:44:25.643+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v01', 'get_age', 'manual__2025-04-01T03:44:24.338278+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpo5kbdu4y']
[2025-04-01T03:44:25.645+0000] {standard_task_runner.py:105} INFO - Job 115: Subtask get_age
[2025-04-01T03:44:25.689+0000] {task_command.py:467} INFO - Running <TaskInstance: our_dag_with_python_operator_v01.get_age manual__2025-04-01T03:44:24.338278+00:00 [running]> on host 9663692256f6
[2025-04-01T03:44:25.784+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rajesh' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v01' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T03:44:24.338278+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T03:44:24.338278+00:00'
[2025-04-01T03:44:25.785+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T03:44:25.823+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-01T03:44:25.832+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T03:44:25.833+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v01, task_id=get_age, run_id=manual__2025-04-01T03:44:24.338278+00:00, execution_date=20250401T034424, start_date=20250401T034425, end_date=20250401T034425
[2025-04-01T03:44:25.869+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T03:44:25.898+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-01T03:44:25.899+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
